.babelrc

```diff
@@ -9,6 +9,7 @@
   "plugins": [
     "dev-expression",
     "add-module-exports",
-    "transform-object-assign"
+    "transform-object-assign",
+    ["transform-react-remove-prop-types", { mode: "wrap" }]
```

Comment:
Yes, correct.  Sample output bundle from development build would look as follows:

```
TransitionGroup.propTypes = process.env.NODE_ENV !== "production" ? {
  component: _propTypes2.default.any,
  childFactory: _propTypes2.default.func,
  children: _propTypes2.default.node
} : {};
```

In the now production build for this app (as well as end-users who import and build production bundles using either Webpack `-p` or equivalently **Define** + **Uglify** plugins) will result in:

**After define**
```
TransitionGroup.propTypes = false ? {
  component: _propTypes2.default.any,
  childFactory: _propTypes2.default.func,
  children: _propTypes2.default.node
} : {};
```

**After uglify **
```
TransitionGroup.propTypes = {};
```